tcms/rpc/api/forms/testrun.py

```diff
@@ -21,39 +25,30 @@ def assign_plan(self, plan_id):
         self.populate(self.fields['plan'].queryset.first().product_id)
 
 
-class UpdateForm(NewForm):
-    plan = forms.ModelChoiceField(
-        queryset=TestPlan.objects.all(),
-        required=False,
-    )
-    summary = forms.CharField(
-        required=False
-    )
-    manager = forms.ModelChoiceField(
-        queryset=User.objects.all(), required=False
-    )
-    product = forms.ModelChoiceField(
-        queryset=Product.objects.all(),
-        empty_label=None, required=False
-    )
-    product_version = forms.ModelChoiceField(
-        queryset=Version.objects.none(),
-        empty_label=None, required=False
-    )
-    build = forms.ModelChoiceField(
-        queryset=Build.objects.all(),
-        required=False
+class UpdateForm(UpdateModelFormMixin, forms.ModelForm):
+
+    manager = UserField()
+    default_tester = UserField(required=False)
```

Comment:
Same comment for this field as before, the `required` attribute isn't really necessary b/c the mixin part takes care of that. Keep the declarations consistent.